src/components/PostForm.tsx

```diff
@@ -0,0 +1,120 @@
+import { h, Component } from 'preact';
+import { Button } from '@mui/material';
+import { newPublicPost, getCurrentAuthor } from '../utils/apiCalls';
+
+type Props = {  };
+type State = { 
+    body: string
+    category: string
+    title: string
+    authorDisplayName: string
+    authorId: number
+};
+
+class PostForm extends Component<Props, State> {
+    constructor() {
+        super();
+        this.state = { 
+            body: "What's on your mind?",
+            category: "",
+            title: "",
+            authorDisplayName: "",
+            authorId: 2
+        };
```

Comment:
what happened with the tags?

Reply:
Tags was the category column, and I noticed that Category is a string in our schema so I updated it to that.